src/_stylesheets/_swatch.scss

```diff
@@ -44,7 +44,7 @@
 
 // Reduce the spacing between swatches in a list
 .app-swatch-list {
-  margin-top: 0;
+  margin-top: govuk-spacing(1);
```

Comment:
Is this doing anything? From a quick tests this seems to collapse into the heading margin that's typically above these swatches. Is there a specific case you're accounting for?

Reply:
This adds 5px on top of the swatches (along with the 10px app-top-border padding) to make 15px of visible spacing between the border line and the swatch. We felt the content was too close to the top border without it.


| **Without 5px margin-top**  | **With 5px margin-top** |
|------------------------------------|--------------------------------------|
| <img width="auto" height="278" alt="Swatch list without 5px margin-top" src="https://github.com/user-attachments/assets/ddeba991-c6cd-4a14-bb2c-7247e60a9732" />         | <img width="531" height="278" alt="Screenshot 2025-09-22 at 13 07 40" src="Swatch list with 5px margin-top" />          |

Reply:
Brill, it was where it's being used I wanted clarity on. Tata.